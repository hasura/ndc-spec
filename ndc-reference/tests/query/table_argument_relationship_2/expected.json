[
  {
    "rows": [
      {
        "first_name": {
          "value": "Peter"
        },
        "last_name": {
          "value": "Landin"
        },
        "articles": {
          "rows": {
            "rows": [
              {
                "id": {
                  "value": 1
                },
                "title": {
                  "value": "The Next 700 Programming Languages"
                }
              }
            ]
          }
        }
      },
      {
        "articles": {
          "rows": {
            "rows": [
              {
                "id": {
                  "value": 2
                },
                "title": {
                  "value": "Why Functional Programming Matters"
                }
              },
              {
                "title": {
                  "value": "The Design And Implementation Of Programming Languages"
                },
                "id": {
                  "value": 3
                }
              }
            ]
          }
        },
        "first_name": {
          "value": "John"
        },
        "last_name": {
          "value": "Hughes"
        }
      }
    ]
  }
]