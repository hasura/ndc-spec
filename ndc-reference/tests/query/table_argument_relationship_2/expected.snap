---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/table_argument_relationship_2/request.json
---
[
  {
    "rows": [
      {
        "first_name": "Peter",
        "last_name": "Landin",
        "articles": {
          "rows": [
            {
              "id": 1,
              "title": "The Next 700 Programming Languages"
            }
          ]
        }
      },
      {
        "first_name": "John",
        "last_name": "Hughes",
        "articles": {
          "rows": [
            {
              "id": 2,
              "title": "Why Functional Programming Matters"
            },
            {
              "id": 3,
              "title": "The Design And Implementation Of Programming Languages"
            }
          ]
        }
      }
    ]
  }
]
